### PR TITLE
Feature/m2 955 workspaces api

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/shared/layouts/BaseLayout/components/LeftBar/LeftBar.tsx [hunk 1]
```diff
@@ -1,27 +1,51 @@
...
+        {
+          owned: true,
+          ownerId: id,
+          workspaceName: `${firstName} ${lastName}`,
+        },
+        ...(result?.data?.result || []),
+      ]);
+  });
+
+  useEffect(() => {
```
Comment (IhorNesterenko99) [inline src/shared/layouts/BaseLayout/components/LeftBar/LeftBar.tsx:37]: For what do you need this useEffect?

File: src/shared/layouts/BaseLayout/components/LeftBar/mocked.ts
[the remark this answers is not in the document]
Comment (kdanilenka32) [reply src/shared/layouts/BaseLayout/components/LeftBar/mocked.ts:1]: It used in other place

File: src/shared/layouts/BaseLayout/components/LeftBar/LeftBar.tsx (hunk 1 above)
Comment (kdanilenka32) [reply src/shared/layouts/BaseLayout/components/LeftBar/LeftBar.tsx:37]: for getWorkspacesApi call

File: src/modules/Dashboard/state/Applets/Applets.reducer.ts [hunk 2]
```diff
@@ -12,15 +13,15 @@ import {
...
   createAppletsPendingData({ builder, thunk: getEvents, key: 'events' });
 
-  createAppletsFulfilledData({ builder, thunk: getApplets, key: 'applets' });
+  createAppletsFulfilledData({ builder, thunk: getWorkspaceApplets, key: 'applets' });
   createAppletsFulfilledData({ builder, thunk: getApplet, key: 'applet' });
   createAppletsFulfilledData({ builder, thunk: getEvents, key: 'events' });
 
-  createAppletsRejectedData({ builder, thunk: getApplets, key: 'applets' });
   createAppletsRejectedData({ builder, thunk: getApplet, key: 'applet' });
+  createAppletsRejectedData({ builder, thunk: getWorkspaceApplets, key: 'applets' });
```
Comment (muhurov) [inline src/modules/Dashboard/state/Applets/Applets.reducer.ts:25]: Is it OK using 'applets' key for workspaceApplets? As I can see there is getAppletsApi

Comment (kdanilenka32) [reply src/modules/Dashboard/state/Applets/Applets.reducer.ts:25]: yes, I set applets

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**